third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -593,21 +676,29 @@ impl InterpreterImpl {
         loader: &Loader,
         function: Rc<LoadedFunction>,
         frame_cache: Rc<RefCell<FrameTypeCache>>,
+        mask: ClosureMask,
+        mut captured: Vec<Value>,
     ) -> PartialVMResult<Frame> {
         let mut locals = Locals::new(function.local_tys().len());
         let num_param_tys = function.param_tys().len();
-
-        for i in 0..num_param_tys {
-            locals.store_loc(
-                num_param_tys - i - 1,
-                self.operand_stack.pop()?,
-                loader.vm_config().check_invariant_in_swap_loc,
-            )?;
+        for i in (0..num_param_tys).rev() {
```

Comment:
You could probably use the same helper function you had in ClosureMask to avoid the for loops here? Simialar to call_native